data/built-ins.json

```diff
@@ -671,28 +712,31 @@
     "edge": 14,
     "firefox": 43,
     "safari": 10,
+    "node": 6,
     "ios": 10,
     "opera": 34
   },
   "es7.object.values": {
     "chrome": 54,
     "edge": 14,
     "firefox": 47,
+    "safari": 10.1,
     "node": 7,
     "opera": 41
   },
   "es7.object.entries": {
     "chrome": 54,
     "edge": 14,
     "firefox": 47,
+    "safari": 10.1,
     "node": 7,
     "opera": 41
   },
   "es7.object.get-own-property-descriptors": {
     "chrome": 54,
     "edge": 15,
     "firefox": 50,
-    "safari": 10,
```

Comment:
This is because of [this](https://github.com/babel/babel-preset-env/blob/master/data/built-in-features.js#L129) matches both tests in [compat-table](https://github.com/kangax/compat-table/blob/gh-pages/data-es2016plus.js#L143-L186) and only one is true in sf10.

This happens because of this logic: https://github.com/babel/babel-preset-env/blob/master/scripts/build-data.js#L89

I think this is okay, as in sf10 it seems to be not implemented completely, but not sure what the intention was here.